core/src/main/resources/hudson/model/Computer/index.jelly

```diff
@@ -28,6 +28,20 @@ THE SOFTWARE.
     <st:include page="sidepanel.jelly" />
     <l:main-panel>
 
+      
+
+      <h1>
+        <span class="icon-lg">
+          <l:icon src="${it.iconClassName}" />
+        </span>
+        ${it.caption}
```

Comment:
I think there is a problem here because the icon and the text are not aligned vertically. I'm trying to find a solution for this.

Reply:
I removed the `<span/>` and added the `icon-*` to the `<l:icon/>` but the class is not passed to the svg (and the icon is 512px by 512px). Adding the class to the resulting SVG is however fixing the icon size.

Reply:
we might have a larger issue with the `symbol` because the class is not passed and I could only fix this with editing the `icon` layout and adding the `icon-md` to https://github.com/jenkinsci/jenkins/blob/0f0a5097ed0a45b78d58e976dd29d32e9f6a62ac/core/src/main/resources/lib/layout/icon.jelly#L71 because the `iconMetadata.classSpec` is always empty.

@janfaracik could you help here? maybe I'm over-thinking the problem here.

Reply:
thx any help are welcome

Reply:
> we might have a larger issue with the `symbol` because the class is not passed and I could only fix this with editing the `icon` layout and adding the `icon-md` to
> 
> https://github.com/jenkinsci/jenkins/blob/0f0a5097ed0a45b78d58e976dd29d32e9f6a62ac/core/src/main/resources/lib/layout/icon.jelly#L71
> 
> because the `iconMetadata.classSpec` is always empty.
> @janfaracik could you help here? maybe I'm over-thinking the problem here.

Hey, I spotted that too (https://github.com/jenkinsci/design-library-plugin/pull/67#issuecomment-1134005433) - changing it to this 

![](https://user-images.githubusercontent.com/43062514/169719059-36746639-755a-4db5-b076-1dd1ff8dd79a.png)

will fix the missing class issue, I've given a quick look around Jenkins and it doesn't seem to have broken anything but I can't guarantee that.

Reply:
@janfaracik  So when I understand it correctly, there are no change request for this PR. This shall be done somewhere else (global) right?

Reply:
@mPokornyETM I still have a problem. Because you added a `<span/>` with the `icon-lg` class, the icon is not aligned correctly. The code should be something like `<h1><l:icon/>Title</h1>` imo.

Reply:
I don`t know, hwy it shows as change, But I haven´t change the part
```
<span class="icon-lg">
  <l:icon src="${it.iconClassName}" />
</span>
```